### DISCUSSION TITLE
Keep dist filenames the same?

### DISCUSSION THREAD

**Discussion by notflip** [quasarframework/quasar#16012] [General - Components / Directives / etc] [answered]
![Screenshot 2023-06-26 at 09 16 24](https://github.com/quasarframework/quasar/assets/4146384/f5fa1774-a1a1-45ca-abc6-6704e35b6711)

Everytime I build, I get 2 new files, the one that is deleted because of the filename change, and the new one. Is it possible to disable this? Or is this feature needed for Quasar to work properly?

Thanks!

**Comment by TobyMosque** [comment]
by default, files inside the dist folder are ignored, so they aren't versionated.
but regarding your question, that random number isn't that random, this is just the file hash.
without that, your users will face cache issues after each release (if u not disable the cache, what isn't a good idea)

**Reply by notflip** [reply]
You're right, I was having build issues on the host server (quasar CLI is not installed there), so the `quasar build` command wasn't working that's why I built it on my local machine, I added the dist folder to gitignore again, I suppose I should just install quasar CLI on the server? Or is it possible to build using npm? without installing the CLI?

**Comment by metalsadman** [comment, marked as answer]
https://github.com/quasarframework/quasar/discussions/14335
https://quasar.dev/quasar-cli-vite/quasar-config-js#build
`build: { useFilenameHashes: false }`

**Reply by notflip** [reply]
Thanks a lot!
